doc/integration_test_report.rst

```diff
@@ -356,11 +356,13 @@ All attributes for items having a non-empty attribute
     :asil: ^.+$
     :sort: asil
 
-.. item-attributes-matrix:: All attributes for items having a non-empty asil+aspice attribute
+.. item-attributes-matrix:: All attributes for items having a non-empty asil+aspice attribute; transposed
```

Comment:
Add new item matrix for transpose option

Reply:
Duplicated this matrix and transposed it to show the effect more clearly 06cc814d83f074d7bb18b76970469cd9d03703c2